### ISSUE TITLE
vg call throwing error from tabix

### ISSUE THREAD

**Issue by jcoliver** [NCBI-Hackathons/Graph_Genomes_CSHL#9]
```
REFFASTA="/home/data/reference/GRCh38_full_plus_hs38d1_analysis_set_minus_alts.fa"
VCFFILE="/home/devsci4/Graph_Genomes_CSHL/examples/vgInput.vcf.gz"
VGOUTFILE="/home/devsci4/Graph_Genomes_CSHL/examples/vgoutput.vg"
vg/bin/vg construct -r $REFFASTA -v $VCFFILE > $VGOUTFILE
```

Returns:

`[tabix++] the index file is older than the vcf file. Please use '-f' to overwrite or reindex.`

**Comment by nhansen** [comment]
tabix –p vcd $VCFFILE

?

—Nancy

**Comment by jcoliver** [comment]
Yep! That's right. Fixed on commit eee79b4 .
